pages/auth.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="page-ctn">
+    <div class="background">
+      <div class="lhs"></div>
+      <div class="rhs"></div>
+    </div>
+    <div class="backdrop">
+      <div class="main-container">
+        <div class="sub-container">
+          <img class="documenter-icon" src="~assets/images/documenter-icon.svg" alt="">
+          <h2 class="welcome-user">Hello Prudence,</h2>
+          <p class="user-details">Set your username and  password</p>
+          <div class="input-container">
+            <input id="username" v-model="username" type="text" placeholder="Username">
+            <input id="password" v-model="password" type="text" placeholder="Password">
+            <input id="update-password" v-model="password" type="text" placeholder="Confirm Password">
+            <button class="submit">Continue</button>
+          </div>   
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    name:"AuthPage",
+    layout:"AuthLayout"
+  }
+</script>
+
+<style scoped>
+.background{
+  height: 100vh;
```

Comment:
setting the height of a container element is bad practice if it houses a lot of elements or when it spans a whole viewport. try `min-height` instead @SualaDev